src/streamlink/plugins/crunchyroll.py

```diff
@@ -305,9 +300,33 @@ def stream_weight(cls, key):
         return Plugin.stream_weight(key)
 
     def _get_streams(self):
-        api = self._create_api()
-        media_id = int(self.match.group("media_id"))
+        beta_id = self.match.group("beta_id")
+        if beta_id:
+            json = self.session.http.get(self.url, schema=validate.Schema(
+                validate.parse_html(),
+                validate.xml_xpath_string(".//script[contains(text(), 'window.__INITIAL_STATE__')]/text()"),
+                validate.any(None, validate.all(
+                    validate.transform(self.beta_json_re.search),
+                    validate.any(None, validate.all(
+                        validate.get(1),
+                        validate.parse_json(),
+                        validate.any(None, validate.all(
+                            {"content": {"byId": {str: {"external_id": validate.all(
+                                validate.transform(lambda s: int(s.replace("EPI.", ""))),
+                                int,
```

Comment:
The `int` type validation is unnecessary, as you're already transforming to `int` above